R/serializer.R

```diff
@@ -231,6 +231,26 @@ serializer_unboxed_json <- function(auto_unbox = TRUE, ..., type = "application/
   serializer_json(auto_unbox = auto_unbox, ..., type = type)
 }
 
+#' @describeIn serializers GeoJSON serializer. See also [geojsonsf::sf_geojson()] and [[geojsonsf::sfc_geojson()]].
+#' @inheritParams geojsonsf::sf_geojson
+#' @inheritParams geojsonsf::sfc_geojson
```

Comment:
No parameters will be matched. Removing param inheritance.

```suggestion
```

---------------------

Maybe in the description on the line above, mention that the `...` will be passed into `geojsonsf::sfc_geosjon()` or `geojsonsf::sf_geojson()` respectively?